db/db.js

```diff
@@ -379,6 +379,24 @@ async function getLocationData(locationid) {
 
 }
 
+// Retrieves the locationid corresponding to a particle device coreID
+
+async function getLocationFromCoreID(coreID){
+    try{
+        const results = await pool.query("SELECT (locationid) FROM locations WHERE door_coreID = $1 OR radar_coreID = $1", [coreID]);
```

Comment:
could you explain this to me a little bit? why this ambiguity between the two types of ID?

Reply:
each locationID uses two argon devices, one for the radar and the other for the door. Each has its own unique coreID